### DISCUSSION TITLE
Immich with Lighttpd reversere proxy

### DISCUSSION THREAD

**Discussion by Loafter** [immich-app/immich#5061] [Q&A]
Hi have issues with reverse proxy using lightttpd, at this moment it's partialy work.

1. Map not load (webgl connection error)
2. Immich alway try to ask login in basic auth in random places.

i have strugles with reinterpet exact value from nginx config to real ip and hostnames
my enviroment:

 

- imich:
- localip 10.9.9.13
- lighttp:
- realip 95.123.222.111
- localt ip 10.9.9.1
- hotname immich.example.net

        proxy_pass http://10.9.9.13:2283;
        proxy_set_header Host              ???????;
        proxy_set_header X-Real-IP        ???????;
        proxy_set_header X-Forwarded-For   ?????;
        proxy_set_header X-Forwarded-Proto https????;

        # http://nginx.org/en/docs/http/websocket.html
        proxy_http_version 1.1;
        proxy_set_header   Upgrade    $http_upgrade;
        proxy_set_header   Connection "upgrade";
        proxy_redirect off

**Comment by gstrauss** [comment]
https://wiki.lighttpd.net/mod_proxy
`proxy.server = ( "" => (( "host" => "10.9.9.13", "port" => 2283 )) )`
`proxy.header += ("upgrade" => "enable")`
